infra/Pos/Slotting/Impl/Simple.hs

```diff
@@ -42,49 +53,68 @@ data SimpleSlottingState = SimpleSlottingState
     { _sssLastSlot :: !SlotId
     }
 
-type SimpleSlottingVar = TVar SimpleSlottingState
+type SimpleSlottingStateVar = TVar SimpleSlottingState
 
-mkSimpleSlottingVar :: (MonadIO m, HasConfiguration) => m SimpleSlottingVar
-mkSimpleSlottingVar = atomically $ newTVar $ SimpleSlottingState $ unflattenSlotId 0
+mkSimpleSlottingStateVar :: (MonadIO m, HasConfiguration) => m SimpleSlottingStateVar
+mkSimpleSlottingStateVar = atomically $ newTVar $ SimpleSlottingState $ unflattenSlotId 0
 
 ----------------------------------------------------------------------------
 -- Implementation
 ----------------------------------------------------------------------------
 
-getCurrentSlotSimple
+getCurrentSlotSimple'
     :: (SimpleSlottingMode ctx m)
-    => SimpleSlottingVar
+    => SimpleSlottingStateVar
     -> m (Maybe SlotId)
-getCurrentSlotSimple var = traverse (updateLastSlot var) =<< (currentTimeSlottingSimple >>= slotFromTimestamp)
+getCurrentSlotSimple' var = do
+    -- var <- view (lensOf @SimpleSlottingStateVar)
+    traverse (updateLastSlot var) =<< (currentTimeSlottingSimple >>= slotFromTimestamp)
```

Comment:
I would prefer for everything to read in a consistent direction -- either

```haskell
traverse (updateLastSlot var) =<< slotFromTimestamp =<< currentTimeSlottingSimple

--or,

slotFromTimestamp
    >>= currentTimeSlottingSimple
    >>= traverse (updateLastSlot var)
```

Switching directions mid-line makes it hard to read.

Reply:
Yes, I shouldn't copy paste it. It's indeed error prone (there is one in your comment with wrong order, which proves the statement :) )